src/components/AppLayout/Header/components/Layout.tsx

```diff
@@ -68,7 +68,7 @@ const Layout = ({ classes, providerDetails, providerInfo }) => {
   const { clickAway: clickAwayNetworks, open: openNetworks, toggle: toggleNetworks } = useStateHandler()
   const networkName = getNetworkName().toLowerCase()
   const networks = getNetworks()
-  const { isDesktop }: { isDesktop: boolean } = (window as unknown) as Partial<Window> & { isDesktop: boolean }
+  const { isDesktop } = window as Window
```

Comment:
you don't need `as Window`. `window` is already `Window`

In the link I shared above we just added `isDesktop` property to an existing interface `Window`. It should work without any type assertion